src/cli_plugin/install/__tests__/download.js

```diff
@@ -0,0 +1,226 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import nock from 'nock';
+import glob from 'glob-all';
+import rimraf from 'rimraf';
+import mkdirp from 'mkdirp';
+import Logger from '../../lib/logger';
+import { download, _downloadSingle } from '../download';
+import { join } from 'path';
+
+describe('kibana cli', function () {
+
+  describe('plugin downloader', function () {
+    const testWorkingPath = join(__dirname, '.test.data');
```

Comment:
Actually, you're probably better off just circumventing the actual file system in your tests altogether.  Something like [mock-fs](https://github.com/tschaub/mock-fs) would allow you to mock away the file system entirely, which means the tests would run a lot faster and you would never run into issues with orphaned artifacts and the like.